modules/virt-about-node-placement-vms.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/install/virt-specifying-nodes-for-vms.adoc
+
+[id="virt-about-node-placement-vms_{context}"]
+= About node placement for virtual machines
+
+To ensure that virtual machines (VMs) run on appropriate nodes, you can configure node placement rules. You might want to do this if:
+
+* You have several VMs. To ensure fault tolerance, you want them to run on different nodes.
+* You have two chatty VMs. To avoid redundant inter-node routing, you want the VMs to run on the same node.
+* Your VMs require specific hardware features that are not present on all available nodes.
+* You have a pod that adds capabilities to a node, and you want to place a VM on that node so that it can use those capabilities.
+
+You can use the following rule types in the `spec` field of a `VirtualMachine` manifest:
+
+`nodeSelector`:: Allows virtual machines to be scheduled on nodes that are labeled with the key-value pair or pairs that you specify in this field. The node must have labels that exactly match all listed pairs.
```

Comment:
IIUC, tables aren't ideal for accessibility unless the info being presented is more complex (such as something that would have 3 columns instead of 2). I might be wrong about this, though!